frontend/app/views/fragments/tier/comparisonTable.scala.html

```diff
@@ -30,30 +33,28 @@
     </div>
 }
 
-@comparisonCheckmark(checkmark: Boolean) = {
-    <div class="comparison-table__checkmark">
+@comparisonCheckmark(tier: Tier, checkmark: Boolean, isActive: Boolean = false) = {
+    <div class="comparison-table__checkmark comparison-table--clickable@if(isActive){ is-active}" data-tier="@tier.name.toLowerCase">
         <div class="comparison-table__checkmark__inner">
         @fragments.common.checkmark(checkmark)
         </div>
     </div>
 }
 
-@comparisonPricing(tier: Tier) = {
-    <div class="comparison-table__trail">
+@comparisonPricing(tier: Tier, isActive: Boolean = false) = {
+    <div class="comparison-table__trail comparison-table--clickable@if(isActive){ is-active}" data-tier="@tier.name.toLowerCase">
```

Comment:
Bit of BEM weirdness here, you've got a parent modifier `comparison-table--clickable` on a child `comparison-table__trail` is there another way of phrasing this selector?